### ISSUE TITLE
GUI: Investigate possible error with high-volume of error messages causing GUI to hang

### ISSUE THREAD

**Issue by AdrianPotter** [ISISComputingGroup/IBEX#1521]
When setting up NDXIRIS test, we created a sample configuration with many IOCs. The system responded adequately, provided there weren't too many IOCs running. At a certain point though, the system became completely unresponsive

**Comment by AdrianPotter** [comment]
This occurred when we were running NDXIRISTEST with a selection of:

Cryovalve
TPG26x
TPG300
Lakeshore336
Lakeshore218
Eurotherm
PDR2000

I suspect the issue arose because of the log server being overwhelmed by too many incoming messages in RECSIM mode. Notably the RECSIM mode for the cryovalve was broken (#1519) leading to many more messages than would arrive otherwise.

The issue may have been fixed with issue #1519 but the GUI should nonetheless stay responsive in the face of a high volume of messages from the server.

**Comment by AdrianPotter** [comment]
Of course, it may be nothing to do with the messages

**Comment by KathrynBaker** [comment]
Started to look at this, but have diverted into the database server issue I shall create, after all, if too many messages cause the system to lock up there is probably a bigger problem on a live system than the lock up at the moment. I ran a test overnight with a number of erroring IOCs as a quick and dirty, and there was a noticeable lag in my system until I did a few things like change to a less demanding config. My next plan was to write a specific test to input messages at different rates to test the message log handling, just in order to try and benchmark what the problem is, and to reproduce the issue more reliably.
